metadata-db-app/int_test/cmr/metadata_db/int_test/concepts/collection_save_test.clj

```diff
@@ -63,7 +55,7 @@
             _ (Thread/sleep 10)
             {second-revision-id :revision-id} (util/save-concept initial-collection)
             _ (Thread/sleep 10)
-            {tombstone-revision-id :revision-id} (util/save-concept {:deleted true :concept-id concept-id})
+            {tombstone-revision-id :revision-id} (util/delete-concept concept-id)
```

Comment:
Hrm, I thought that these two were identical (I'll look again) ... which led me to believe that `delete-concept` was newer and the test hadn't been updated yet.